core/src/com/example/simulation/action/UpdateHealthAction.java

```diff
@@ -0,0 +1,22 @@
+package com.example.simulation.action;
+
+public class UpdateHealthAction extends TeamAction{
+
+    int newHealth;
```

Comment:
Gibt es einen Grund, warum das hier nicht `private final` wie in anderen Actions ist?